som-interpreter-ast/src/main.rs

```diff
@@ -21,22 +21,22 @@ use som_interpreter_ast::value::Value;
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
 
-#[derive(Debug, Clone, PartialEq, StructOpt)]
+#[derive(Debug, Clone, PartialEq, clap::StructOpt)]
 #[structopt(about, author)]
 struct Options {
     /// Files to evaluate.
-    #[structopt(name = "FILE")]
+    #[clap(name = "FILE")]
     file: Option<PathBuf>,
 
-    #[structopt(name = "ARGS")]
+    #[clap(name = "ARGS")]
     args: Vec<String>,
 
     /// Set search path for application classes.
-    #[structopt(short, long)]
+    #[clap(short, long)]
```

Comment:
`clap` made a breaking change for users coming from `structopt` by changing `Vec<T>` to use `multiple_occurences(true)` instead of `multiple_values(true)`, which means that, by default, the `-c val1 val2 val3` pattern stops working and is instead replaced by the `-c val1 -c val2 -c val3` pattern.

So, we need to add `multiple_values(true)` to restore the old behaviour and avoid breakages in most testing and benchmarking setups out there (like it did for the GitHub CI here).

```suggestion
    #[clap(short, long, multiple_values(true))]
```